src/pages/api/bookings/GET.ts

```diff
@@ -0,0 +1,19 @@
+import connectMongo from "../../../util/connectMongo";
+import Booking from "../../../../models/booking";
+
+/**
+ * @param {import('next').NextApiRequest} req
+ * @param {import('next').NextApiResponse} res
+ */
+
+export default async function getBookings(req: any, res: any) {
```

Comment:
I believe you're mixing up the patterns from the app router and the pages router here (and in all other API resources in this PR). The `GET` file name has no special meaning. It just gets interpreted as a part of the path, meaning that this function gets invoked when making a request to localhost:3000/api/bookings/GET, _regardless of HTTP method_.

My guess is that you wanted this function to map to `GET /api/bookings`, i.e. when making a `GET` request to the `/api/bookings` URI, but that's not what it does.